pkg/siftool/testdata/TestAddCommands/Add/out.golden

```diff
@@ -5,7 +5,7 @@ Usage:
 
 Examples:
 siftool add image.sif recipe.def -datatype 1
-siftool add image.sif rootfs.squashfs --datatype 4 --parttype 1 --partfs 1 ----partarch 2
+siftool add image.sif rootfs.squashfs --datatype 4 --parttype 1 --partfs 1 --partarch 2
```

Comment:
It looks like the examples in the line before/after this have the same problem? Could you update those too?